src/test/attributes.ts

```diff
@@ -77,11 +77,11 @@ describe('attributes', function () {
       assert.strictEqual(elm.hasAttribute('required'), false);
       assert.strictEqual(elm.getAttribute('required'), null);
     });
-    it('is not omitted if the value is falsy but casted to string', function () {
+    it('is not omitted if the value is falsy', function () {
       var vnode1 = h('div', { attrs: { readonly: 0, noresize: null as any } });
       elm = patch(vnode0, vnode1).elm;
-      assert.strictEqual(elm.getAttribute('readonly'), '0');
-      assert.strictEqual(elm.getAttribute('noresize'), 'null');
+      assert.ok(elm.hasAttribute('readonly'));
+      assert.ok(elm.hasAttribute('noresize'));
```

Comment:
Casting to a string is most likely performed by the platform. And since IE11 seems to keep the `null` as a real `null`, it seems enough for us to test for the presence of the boolean attribute. A boolean attribute with `null` as a value is still present (truthy), right?

Reply:
So `elm.getAttribute('noresize')` is `null` instead of `'null'` in IE11. What about the first line?

Reply:
`hasAttribute` is all that's necessary for a boolean attribute, I figure.